Cargo.toml

```diff
@@ -58,7 +58,7 @@ hex = { version = "0.4.3", default-features = false }
 hmac = { version = "0.12.1", default-features = false }
 itertools = { version = "0.10.3", default-features = false }
 # TODO: don't activate `static-context` by default; see https://github.com/paritytech/libsecp256k1/issues/77
```

Comment:
@tomaka can we remove this comment since a) the issue is closed b) the reason we're not enabling default features is no-std, no?

Reply:
Yes

Reply:
I don't think a new version has been released though